CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# How to contribute
+
+To contribute to Pomona, there's just a few things you need to know.
+
+## Getting Started
+
+* Read and make sure you agree with the [Code of Conduct][coc].
+* Make sure you have a [GitHub account][github].
+* Then, you have three options:
+    1. Submit a ticket for your issue, assuming one does not already exist.
+        * Clearly describe the issue including steps to reproduce when it is a bug.
+        * Make sure you fill in the earliest version that you know has the issue.
+    2. [Directly edit the file][edit] you want to change on GitHub.
+    3. [Fork the repository on GitHub][forking].
+
+If you choose option 3 (forking the repository), then please read on.
+
+## Making Changes
+
+* Create a new [branch][branching] from where you want to base your work.
+  * This is usually the `master` branch.
+  * Please avoid working directly on the `master` branch.
```

Comment:
I think the `develop` branch would be a better base for this, since we aim to keep `develop` mostly green anyway..